parser_extensions_test.go

```diff
@@ -0,0 +1,69 @@
+package protokit_test
+
+import (
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/protoc-gen-go/descriptor"
+)
+
+func registerTestExtensions() {
```

Comment:
Seems like this is only used in parser_test.go. Should we just move it there?

Reply:
Fine with me. Will do